### PR TITLE
#167165046 A chore to setup postgres

### PR DESCRIPTION
**Pull request by Tunji17**
### What does this PR do?
Changes the database from mongo to Postgres
#### Description of Task to be completed?
Used sequelize ORM to interface with a Postgres database for both test and development
#### How should this be manually tested?

> clone this repo using git clone 
> and run `npm run migrate`

#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/167165046
#### Screenshots (if appropriate)
https://drive.google.com/file/d/19qonsai9Nd24CH1IbUlCpwrqmktKwuyQ/view?usp=sharing

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,2 @@
+DATABASE_URL = postgres://mazus:mazusgeneral@localhost:5432/mazusdevdb
+TEST_DATABASE_URL = postgres://mazustest:mazusgeneraltest@localhost:5432/mazustestdb
```
Comment (mazma1) [inline .env]: Did you push .env to Github?

File: database/models/articleimage.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+      allowNull: false,
+      type: DataTypes.STRING
+    },
+    articleId: {
+      allowNull: false,
+      type: DataTypes.INTEGER
+    },
+  }, {});
+  ArticleImage.associate = function(models) {
+    // associations can be defined here
```
Comment (tolulope-od) [inline database/models/articleimage.js:17]: Hi, please is there any reason why there isn't an association between `ArticleImage` and `Articles` table?

**Review by tolulope-od — COMMENTED** [review]
This is was a huge task Tunji. Well done!

File: database/models/articleimage.js (hunk 2 above)
Comment (ocranbillions) [reply database/models/articleimage.js:17]: He implemented it bro... its defined on the Articles model, do check it out.

Comment (tolulope-od) [reply database/models/articleimage.js:17]: Okay thanks for pointing that out, I just needed some clarity because I saw a comment in the file

**Pull request closed without merging**